### ISSUE TITLE
Support Microsoft Edge browser

### ISSUE THREAD

**Issue by SpanishBrowne** [persistory/browserparrot-issues#19]
~/Library/Application Support/Microsoft Edge/Default/History

**Comment by iansinnott** [comment]
Ah good call. Didn't realize it worked on Mac

**Comment by SpanishBrowne** [comment]
Dang.. just realised Edge Profiles stores history in different folders. 

e.g. i have a "work" profile which resides in ~/Library/Application Support/Microsoft Edge/Default/History. Then a "personal" profile that lives at ~/Library/Application Support/Microsoft Edge/Profile 1/History

Ideally want to use BrowserParrot against my personal profile (or maybe both!?). Would be great if user could nominate the location(s)

**Comment by iansinnott** [comment]
Yeah what it does right now is search all profiles. Based on those filepaths you mentioned it looks like it edge is storing history in exactly the same way that Chrome does, and for Chrome we're pulling history for all profiles.

Ideally you could choose which profiles to store history for, but that's not currently implemented. For now all history is indexed.

**Comment by iansinnott** [comment]
This is supported in the rewritten CLI: https://github.com/iansinnott/browser-gopher/blob/master/pkg/extractors/extractors.go#L77-L84

There is not yet an accompanying GUI though, so it's not a drop-in replacement for the current browser parrot yet.
